README.md

```diff
@@ -154,6 +154,49 @@ const suite = new EcdsaSepc256k1Signature2019({
 });
 ```
 
+### Custom documentLoader
+
+Pre-requisites:
+
+* You have an existing valid json-ld @context or json schema.
```

Comment:
```suggestion
* You have an existing valid JSON-LD `@context` or JSON schema.
```

Reply:
This statement doesn't make any sense. A JSON-LD `@context` is a totally different technology from JSON schema and there is no "either or" here. A VC needs to have its own JSON-LD `@context` and it may optionally reference a JSON schema for validating its structure.